### PR TITLE
Multiline for ExpensiTextInput

### LINKED ISSUE #4625: [Hold for payment Sept 8th] Workspace invite - "Add personal message" box in workspace looks very small
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch the app
2. Log in with expensifail account
3. Click on Plus button and select New Workspace
4. Select People
5. Click Invite
6. Put any email addresses or phone number
7. Start typing anything in "Add a personal message" box

## Expected Result:
Add personal message" box in workspace should looks much bigger, so people can able to use scroller and see the all message.

## Actual Result:
Add personal message" box in workspace looks very small. It's uncomfortable to edit a long message. 

## Workaround:
N/A

## Platform:

Where is this issue occurring?

- iOS
- Android

**Version Number:** 1.0.85-0

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/44479856/129264123-af8d0459-6164-45bd-b649-a8be94398e7f.mp4

image

**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### PR DESCRIPTION
**Pull request by kakajann**
### Fixed Issues
Fixes #4625 

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
Screen Shot 2021-08-13 at 10 50 56 PM

#### Mobile Web
Screen Shot 2021-08-13 at 11 17 37 PM

#### iOS
Screen Shot 2021-08-13 at 11 13 19 PM

#### Android
Screenshot_20210813-224941_New Expensify

### CODE REVIEW

**Comment by parasharrajat** [comment]
Looks good. One thing I noticed is that the Copy for email Input is wrong. It accepts multiple comma-separated emails or phone but the text says `Invite a colleague to your workspace`?

**Comment by kakajann** [comment]
I didn't modify anything related email input. This PR's only concern is making new input work with `multiline` prop.

**Comment by parasharrajat** [comment]
> I didn't modify anything related to email input. 

I know. I was just asking maybe a correction can be made, as it's just text. Anyways, it's fine.

**Comment by kakajann** [comment]
@parasharrajat oh, sorry, I misunderstood 😅

I don't mind changing the text as collaborators suggestion

**Comment by shawnborton** [comment]
At first glance, it seems like the gap between the textarea label and the value is too large. Here is a regular text input for reference:
image

And from what I see above:

But maybe that's just my eyes playing tricks on me?

Otherwise can you confirm that the value of the color matches the same text color as text inputs? It appears to be black in the screenshots:
image

**Comment by kakajann** [comment]
- Input color was black, changed to `themeColors.text` (#0b1b34)
- There was a small extra gap between label and input (and I guess it only appears on iOs, maybe because it doesn't support `textAlignVertical="top"`), so I solved using `spacing.pv0`.

Also iOs doesn't support `numberOfLines` prop and on iOs, input height will be as high as its content.

### Here is the result
Screen Shot 2021-08-17 at 10 39 15 AM

**Comment by shawnborton** [comment]
The default height of the textarea feels a bit too tall IMO:
image


Could we make that shorter?

**Comment by kakajann** [comment]
Sure, I'll update in an hour

**Comment by AndrewGable** [comment]
@shawnborton - Anything else you'd like to see here?

File: ios/NewExpensify.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -879,7 +879,7 @@
 				CURRENT_PROJECT_VERSION = 3;
 				DEVELOPMENT_TEAM = 368M544MTT;
 				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;
-				INFOPLIST_FILE = "NewExpensify/Info.plist";
+				INFOPLIST_FILE = NewExpensify/Info.plist;
```
Comment (AndrewGable) [inline ios/NewExpensify.xcodeproj/project.pbxproj:882]: Is this required or related to your change?

File: ios/Podfile.lock [hunk 2]
```diff
@@ -877,7 +877,7 @@ SPEC CHECKSUMS:
   DoubleConversion: cf9b38bf0b2d048436d9a82ad2abe1404f11e7de
   EXHaptics: 337c160c148baa6f0e7166249f368965906e346b
   FBLazyVector: 7b423f9e248eae65987838148c36eec1dbfe0b53
-  FBReactNativeSpec: 884d4cc2b011759361797a4035c47e10099393b5
+  FBReactNativeSpec: c783a75db87c963c60afcd461fc38358805fe5ba
```
Comment (AndrewGable) [inline ios/Podfile.lock:880]: Is this required or related to your change?

**Comment by kakajann** [comment]
@AndrewGable code changes in ios folder is not required neither related. I think `Pod` automatically updated. Do u want me to revert it?

**Comment by AndrewGable** [comment]
Can you confirm these changes happen on `main`? If so, yes we can commit them. Otherwise let's not include them in the branch.

**Comment by shawnborton** [comment]
@kakajann mind updating your screenshots so I can see if my requested changes were applied? Thanks!

**Comment by kakajann** [comment]
Latest change: 5 lines
Screen Shot 2021-08-25 at 12 19 10 AM

**Comment by shawnborton** [comment]
Looks good, thanks!

**Review by AndrewGable — APPROVED** [review]

**Comment by botify** [comment]
This has been deployed to production and is now subject to a 7-day regression period.
If no regressions arise, payment will be issued on 2021-09-08.  :confetti_ball:
